notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,1024 @@
+{
```

Comment:
<p>The assumption of having the first instance doesn't seem right. Maybe just use <code>tensorboard.resource_name</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>done</p>